plugins/available/base.plugin.bash

```diff
@@ -121,7 +121,7 @@ function usage ()
     param '1: directory name'
     group 'base'
     if [ $(uname) = "Darwin" ]; then
-        if [ -n $1 ]; then
+        if [ -n "$1" ]; then
             du -hd $1
```

Comment:
It might make sense to add quotes here as well, just in case it's a directory where the name contains spaces:

```bash
du -hd "$1"
```

Same in lines 131 and 132, I think...